app/models/article.rb

```diff
@@ -466,4 +466,18 @@ def self.time_delta(year = nil, month = nil, day = nil)
     to = to - 1 # pull off 1 second so we don't overlap onto the next day
     return from..to
   end
+
+  def merge_with(other_article_id)
+    # find each of the comments associated w/ the article
+    Article.find(other_article_id).comments.each do |comment|
+      # assign their article id to the id of the current article
+      comment.article_id = self.id
+      comment.save
+    end
+    merged_article = Article.find(other_article_id)
```

Comment:
Looks like your'e doing `Article.find` multiple times. How can you simplify so that it doesn't call the DB multiple times?